packages/augur-ui/src/modules/trading/components/form/form.styles.less

```diff
@@ -150,13 +152,13 @@
   }
 }
 
-.TradingForm__outcome__container__yes {
+.Yes {
   background-color: @color-theme-primary;
   border: 1px solid @color-primary-alternate;
   border-radius: @border-radius-default;
   font-size: @size-14;
```

Comment:
Same comment as above, replace with `.text-14-medium` (also make that var if it doesn't exist yet in `redesign_typography`) here instead of directly modifying font-size/font-weight